app/routes/wallet/sd_jws.py

```diff
@@ -23,16 +23,45 @@
     "/sign",
     response_model=SDJWSCreateResponse,
     summary="Sign SD-JWS",
-    description="""
-Sign Select Disclosure for JWS (SD-JWS)
-
-See https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-07.html for the SD-JWT / SD-JWS spec.
-""",
 )
 async def sign_sd_jws(
     body: SDJWSCreateRequest,
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> SDJWSCreateResponse:
+    """
+    Sign Select Disclosure for JWS (SD-JWS)
+    ---
+
+    The difference between the did and verification_method fields is
+    that if the `did` field is used, acapy will make an educated guess
+    about which key associated with the did to use to sign the jwt.
+
+    While with the `verification_method` field, the user is explicitly
+    specifying which key to use to sign the jwt.
+
+    The `non_sd_list` field is a list of non-selective disclosure attributes.
+    These are attributes that are not included in the selective disclosure i.e.
+    they are always disclosed.
+
+    The endpoint will return the signed SD-JWS with some extra keys needed to reveal the
+    selective disclosure attributes.
+
+    See https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-07.html
+    for the SD-JWT / SD-JWS spec.
+
+    Request body:
+    ---
+        SDJWSCreateRequest: The SD-JWS to sign.
+            did: str: The DID to sign the SD-JWS with.
+            header: dict: The header of the SD-JWS.
+            payload: dict: The payload of the SD-JWS.
+            verification_method: str: The verification method (did with key to use) to use.
+            non_sd_list: list: List of non-selective disclosure attributes.
```

Comment:
btw, headers with an s
and may be worth showing an example request body here as well.
`non_sd_list` is an optional list